src/transformers/models/ctrl/configuration_ctrl.py

```diff
@@ -96,11 +96,6 @@ def __init__(
         embd_pdrop=0.1,
         layer_norm_epsilon=1e-6,
         initializer_range=0.02,
-        summary_type="cls_index",
```

Comment:
These may not appear in modeling file, but they are necessary if the model use
```python
self.sequence_summary = SequenceSummary(config)
```
For `CTRL` model (and some other models below), they don't use `SequenceSummary`. So remove those attributes here.

cc @NielsRogge